cmd/demo/node.go

```diff
@@ -327,7 +341,10 @@ func (n *node) Handle(req *client.ChannelProposal, res *client.ProposalResponder
 		n.handleFinal(p)
 	})
 	go p.ch.ListenUpdates()
-	fmt.Println("\n🆕 Channel opened with", alias, " initial balance:", req.InitBals.Balances[0])
+
+	bals := req.InitBals.Balances[0]
+	myBalEth, peerBalEth := weiToEther(bals[_ch.Idx()]).String(), weiToEther(bals[1-_ch.Idx()]).String()
+	fmt.Printf("\n🆕 Channel opened with %s. Initial balance: [My: %s Ξ, Peer: %s Ξ]\n", alias, myBalEth, peerBalEth)
```

Comment:
If you're working with `Stringer`s, use `%v` in format printing.
```suggestion
	myBalEth, peerBalEth := weiToEther(bals[_ch.Idx()]), weiToEther(bals[1-_ch.Idx()])
	fmt.Printf("\n🆕 Channel opened with %s. Initial balance: [My: %v Ξ, Peer: %v Ξ]\n", alias, myBalEth, peerBalEth)
```
also, use variadic `weiToEth` here.

Reply:
*big.Float doesn't implemet Stringer interface, String() should be called explicitly. See [here](https://golang.org/pkg/math/big/#Float.String).

Reply:
I think it does work, see other comment and https://play.golang.org/p/h_KUgOq_iY2

Reply:
Addressed in this commit - 806b792.